json/twitter-index-pattern.json

```diff
@@ -4,7 +4,7 @@
                 "release_date": "2020-03-02T14:54:52.691488",
```

Comment:
Please update this date too in order to track the version of the index pattern when deployed.

More info about this can be found at: chaoss/grimoirelab-kidash@a0e6b75